namespaces/live.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/main.tf

```diff
@@ -5,15 +5,36 @@ terraform {
 
 provider "aws" {
   region = "eu-west-2"
+
+  default_tags {
+    tags = {
+      # see https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/accessing-the-cloud-console.html
+      GithubTeam = var.team_name
```

Comment:
Folks will need to be in the GitHub team `prison-visits-booking` in order to query AWS CLI for stats. See [Slack thread](https://mojdt.slack.com/archives/C57UPMZLY/p1656419337523929) for info